assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_assignment.vm

```diff
@@ -46,7 +46,7 @@
 					$tlang.getString("gen.due")
 				</th>
 				<td>
-                    #if ($assignment.HideDueDate || $!allowUpdateAssignment)
+                    #if (!$assignment.HideDueDate || $!allowUpdateAssignment)
```

Comment:
if hideDueDate is false then show the dueDate?

Reply:
Correct, HideDueDate = true means don't show the Due Date, so false is show the Due Date.